digits/dataset/generic/test_views.py

```diff
@@ -1,13 +1,16 @@
 # Copyright (c) 2016, NVIDIA CORPORATION.  All rights reserved.
 from __future__ import absolute_import
 
-import json
-
 from bs4 import BeautifulSoup
+import json
+import numpy as np
```

Comment:
Major nitpick here, but PEP8 likes to separate standard library imports and 3rd party imports:

> Imports should be grouped in the following order:
1. standard library imports
2. related third party imports
3. local application/library specific imports
You should put a blank line between each group of imports. 
https://www.python.org/dev/peps/pep-0008/#imports

I've been trying to follow that format in our code since https://github.com/NVIDIA/DIGITS/pull/501.